packages/libs/react-ui/src/styles/vars.css.ts

```diff
@@ -295,7 +324,22 @@ export const darkThemeClass = createTheme(vars.colors, {
   $neutral4: colorPalette.$gray40,
   $neutral5: colorPalette.$gray20,
   $neutral6: colorPalette.$gray10,
-});
+};
+
+// here we combine the old and new values for the dark theme to export only one class
+// eslint-disable-next-line @kadena-dev/typedef-var
+const darkContract = {
+  new: tokens.kda.foundation.color,
+  old: vars.colors,
+};
+export const darkThemeClass = createTheme(
+  darkContract,
+  {
+    new: darkThemeValues.kda.foundation.color,
+    old: oldDarkThemeColors,
+  },
+  'dark',
+);
```

Comment:
So it looks like you're referencing the vars created from the original "old" and "updated" contracts in a new contract that only updates the colors via one classname. Is that correct?